Getting error during initiating a contract negotiation

# Bug Report

## Describe the Bug
C:\edc-sample\Samples>curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @transfer/transfer-01-file-transfer/contractoffer.json "http://localhost:9192/management/v2/contractnegotiations"

[{"message":"There was a problem encountered loading a remote context [code=LOADING_REMOTE_CONTEXT_FAILED].","type":"InvalidRequest","path":null,"invalidValue":null}]
C:\edc-sample\Samples>

### Expected Behavior
Should receive UUID

### Observed Behavior
got error in consumer connector:

JsonLdError[code=There was a problem encountered loading a remote context [code=LOADING_REMOTE_CONTEXT_FAILED]., message=There was a problem encountered loading a remote context [code=LOADING_REMOTE_CONTEXT_FAILED].]
        at com.apicatalog.jsonld.context.ActiveContextBuilder.fetch(ActiveContextBuilder.java:539)
        at com.apicatalog.jsonld.context.ActiveContextBuilder.create(ActiveContextBuilder.java:174)
        at com.apicatalog.jsonld.expansion.ObjectExpansion.initLocalContext(ObjectExpansion.java:200)
        at com.apicatalog.jsonld.expansion.ObjectExpansion.expand(ObjectExpansion.java:102)
        at com.apicatalog.jsonld.expansion.Expansion.compute(Expansion.java:119)
        at com.apicatalog.jsonld.expansion.ObjectExpansion1314.expand(ObjectExpansion1314.java:946)
        at com.apicatalog.jsonld.expansion.ObjectExpansion.expand(ObjectExpansion.java:121)
        at com.apicatalog.jsonld.expansion.Expansion.compute(Expansion.java:119)
        at com.apicatalog.jsonld.expansion.ObjectExpansion1314.expand(ObjectExpansion1314.java:946)
        at com.apicatalog.jsonld.expansion.ObjectExpansion.expand(ObjectExpansion.java:121)
        at com.apicatalog.jsonld.expansion.Expansion.compute(Expansion.java:119)
        at com.apicatalog.jsonld.processor.ExpansionProcessor.expand(ExpansionProcessor.java:124)
        at com.apicatalog.jsonld.api.ExpansionApi.get(ExpansionApi.java:131)
        at org.eclipse.edc.jsonld.TitaniumJsonLd.expand(TitaniumJsonLd.java:68)
        at org.eclipse.edc.connector.api.management.contractnegotiation.ContractNegotiationApiController.initiateContractNegotiation(ContractNegotiationApiController.java:140)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:134)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:177)
        at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:81)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:478)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:400)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:81)
        at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:261)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
        at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)
        at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:240)
        at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:697)
        at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)
        at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346)
        at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:357)
        at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:311)
        at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:764)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:529)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:221)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1381)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:176)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:484)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:174)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1303)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:129)
        at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:192)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)
        at org.eclipse.jetty.server.Server.handle(Server.java:563)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$0(HttpChannel.java:505)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:762)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:497)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:282)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100)
        at org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:969)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1194)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1149)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: JsonLdError[code=The document could not be loaded or parsed [code=LOADING_DOCUMENT_FAILED]., message=URL scheme [http] is not supported.]
        at com.apicatalog.jsonld.loader.SchemeRouter.loadDocument(SchemeRouter.java:59)
        at org.eclipse.edc.jsonld.TitaniumJsonLd$CachedDocumentLoader.loadDocument(TitaniumJsonLd.java:147)
        at com.apicatalog.jsonld.context.ActiveContextBuilder.fetch(ActiveContextBuilder.java:535)
        ... 63 more

## Steps to Reproduce
Steps to reproduce the behavior:
1. Run provider connector
2. Run consumer connector
3. curl command to initiate negotiation
4. See error

## Context Information
_Add any other context about the problem here._

- Used version [e.g. EDC v1.0.0]
- OS: [e.g. iOS, Windows]
- ...

## Detailed Description
_If applicable, add screenshots and logs to help explain your problem._

## Possible Implementation
_You already know the root cause of the erroneous state and how to fix it? Feel free to share your thoughts._

--- a/transfer/transfer-01-file-transfer/contractoffer.json
+++ b/transfer/transfer-01-file-transfer/contractoffer.json
@@ -1,6 +1,7 @@
 {
   "@context": {
-    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+    "edc": "https://w3id.org/edc/v0.0.1/ns/",
+    "odrl": "http://www.w3.org/ns/odrl/2/"
   },
   "@type": "NegotiationInitiateRequestDto",
   "connectorId": "provider",
@@ -12,13 +13,12 @@
     "offerId": "1:test-document:3a75736e-001d-4364-8bd4-9888490edb58",
     "assetId": "test-document",
     "policy": {
-      "@context": "http://www.w3.org/ns/odrl.jsonld",
       "@id": "1:test-document:13dce0f1-52ed-4554-a194-e83e92733ee5",
       "@type": "set",
-      "permission": [],
-      "prohibition": [],
-      "obligation": [],
-      "target": "test-document"
+      "odrl:permission": [],
+      "odrl:prohibition": [],
+      "odrl:obligation": [],
+      "odrl:target": "test-document"
     }
   }
 }

--- a/transfer/transfer-01-file-transfer/filetransfer.json
+++ b/transfer/transfer-01-file-transfer/filetransfer.json
@@ -4,12 +4,9 @@
   },
   "@type": "TransferRequestDto",
   "dataDestination": {
-    "@type": "DataAddress",
     "type": "File",
-    "properties": {
-      "path": "{path to destination file}",
-      "keyName": "keyName"
-    }
+    "path": "{path to destination file}",
+    "keyName": "keyName"
   },
   "protocol": "dataspace-protocol-http",
   "assetId": "test-document",

--- a/transfer/transfer-04-open-telemetry/filetransfer.json
+++ b/transfer/transfer-04-open-telemetry/filetransfer.json
@@ -3,11 +3,9 @@
   "assetId": "test-document",
   "contractId": "{agreement ID}",
   "dataDestination": {
-    "properties": {
-      "path": "/samples/output-file.txt",
-      "keyName": "keyName",
-      "type": "File"
-    }
+    "path": "/samples/output-file.txt",
+    "keyName": "keyName",
+    "type": "File"
   },
   "transferType": {
     "contentType": "application/octet-stream",

--- a/transfer/transfer-05-file-transfer-cloud/README.md
+++ b/transfer/transfer-05-file-transfer-cloud/README.md
@@ -158,12 +158,9 @@ curl --location --request POST 'http://localhost:9192/management/transferprocess
   "assetId": "1",
   "contractId": "<ContractAgreementId>",
   "dataDestination": {
-    "properties": {
-      "type": "AmazonS3",
-      "region": "us-east-1",
-      "bucketName": "<Unique bucket name>"
-    },
-    "type": "AmazonS3"
+    "type": "AmazonS3",
+    "region": "us-east-1",
+    "bucketName": "<Unique bucket name>"
   },
   "managedResources": true,
   "transferType": {

--- a/transfer/transfer-06-consumer-pull-http/README.md
+++ b/transfer/transfer-06-consumer-pull-http/README.md
@@ -181,11 +181,9 @@ curl -d '{
              }
            },
            "dataAddress": {
-             "properties": {
-               "name": "Test asset",
-               "baseUrl": "https://jsonplaceholder.typicode.com/users",
-               "type": "HttpData"
-             }
+             "type": "HttpData",
+             "name": "Test asset",
+             "baseUrl": "https://jsonplaceholder.typicode.com/users"
            }
          }' -H 'content-type: application/json' http://localhost:19193/management/v2/assets \
          -s | jq
@@ -205,15 +203,15 @@ This means that the consumer connector can request any asset from the provider c
 ```bash
 curl -d '{
            "@context": {
-             "edc": "https://w3id.org/edc/v0.0.1/ns/"
+             "edc": "https://w3id.org/edc/v0.0.1/ns/",
+             "odrl": "http://www.w3.org/ns/odrl/2/"
            },
            "@id": "aPolicy",
            "policy": {
-             "@context": "http://www.w3.org/ns/odrl.jsonld",
              "@type": "set",
-             "permission": [],
-             "prohibition": [],
-             "obligation": []
+             "odrl:permission": [],
+             "odrl:prohibition": [],
+             "odrl:obligation": []
            }
          }' -H 'content-type: application/json' http://localhost:19193/management/v2/policydefinitions \
          -s | jq
@@ -344,7 +342,8 @@ send counter offers in addition to just confirming or declining an offer.
 ```bash
 curl -d '{
   "@context": {
-    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+    "edc": "https://w3id.org/edc/v0.0.1/ns/",
+    "odrl": "http://www.w3.org/ns/odrl/2/"
   },
   "@type": "NegotiationInitiateRequestDto",
   "connectorId": "provider",
@@ -356,13 +355,12 @@ curl -d '{
    "offerId": "1:assetId:902a192a-dcc6-49d0-8c9f-17daa9303730",
    "assetId": "assetId",
    "policy": {
-     "@context": "http://www.w3.org/ns/odrl.jsonld",
      "@id": "1:assetId:902a192a-dcc6-49d0-8c9f-17daa9303730",
      "@type": "Set",
-     "permission": [],
-     "prohibition": [],
-     "obligation": [],
-     "target": "assetId"
+     "odrl:permission": [],
+     "odrl:prohibition": [],
+     "odrl:obligation": [],
+     "odrl:target": "assetId"
    }
   }
 }' -X POST -H 'content-type: application/json' http://localhost:29193/management/v2/contractnegotiations \
@@ -450,7 +448,6 @@ curl -X POST "http://localhost:29193/management/v2/transferprocesses" \
         "managedResources": false,
         "protocol": "dataspace-protocol-http",
         "dataDestination": { 
-          "@type": "DataAddress",
           "type": "HttpProxy" 
         }
     }' \

--- a/transfer/transfer-07-provider-push-http/README.md
+++ b/transfer/transfer-07-provider-push-http/README.md
@@ -143,11 +143,9 @@ curl -d '{
              }
            },
            "dataAddress": {
-             "properties": {
-               "name": "Test asset",
-               "baseUrl": "https://jsonplaceholder.typicode.com/users",
-               "type": "HttpData"
-             }
+             "name": "Test asset",
+             "baseUrl": "https://jsonplaceholder.typicode.com/users",
+             "type": "HttpData"
            }
          }' -H 'content-type: application/json' http://localhost:19193/management/v2/assets \
          -s | jq
@@ -167,15 +165,15 @@ This means that the consumer connector can request any asset from the provider c
 ```bash
 curl -d '{
            "@context": {
-             "edc": "https://w3id.org/edc/v0.0.1/ns/"
+             "edc": "https://w3id.org/edc/v0.0.1/ns/",
+             "odrl": "http://www.w3.org/ns/odrl/2/"
            },
            "@id": "aPolicy",
            "policy": {
-             "@context": "http://www.w3.org/ns/odrl.jsonld",
              "@type": "set",
-             "permission": [],
-             "prohibition": [],
-             "obligation": []
+             "odrl:permission": [],
+             "odrl:prohibition": [],
+             "odrl:obligation": []
            }
          }' -H 'content-type: application/json' http://localhost:19193/management/v2/policydefinitions \
          -s | jq
@@ -304,7 +302,8 @@ send counter offers in addition to just confirming or declining an offer.
 ```bash
 curl -d '{
   "@context": {
-    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+    "edc": "https://w3id.org/edc/v0.0.1/ns/",
+    "odrl": "http://www.w3.org/ns/odrl/2/"
   },
   "@type": "NegotiationInitiateRequestDto",
   "connectorId": "provider",
@@ -316,13 +315,12 @@ curl -d '{
    "offerId": "1:assetId:902a192a-dcc6-49d0-8c9f-17daa9303730",
    "assetId": "assetId",
    "policy": {
-     "@context": "http://www.w3.org/ns/odrl.jsonld",
      "@id": "1:assetId:902a192a-dcc6-49d0-8c9f-17daa9303730",
      "@type": "Set",
-     "permission": [],
-     "prohibition": [],
-     "obligation": [],
-     "target": "assetId"
+     "odrl:permission": [],
+     "odrl:prohibition": [],
+     "odrl:obligation": [],
+     "odrl:target": "assetId"
    }
   }
 }' -X POST -H 'content-type: application/json' http://localhost:29193/management/v2/contractnegotiations \
@@ -409,11 +407,8 @@ curl -X POST "http://localhost:29193/management/v2/transferprocesses" \
         "managedResources": false,
         "protocol": "dataspace-protocol-http",
         "dataDestination": { 
-          "@type": "DataAddress",
           "type": "HttpData",
-          "properties": {
-            "baseUrl": "http://localhost:4000/api/consumer/store"
-          }
+          "baseUrl": "http://localhost:4000/api/consumer/store"
         }
     }' \
     -s | jq

